OddNumberSum.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+int main()
+
+{
+   int list[] = { 7, 4, 1, 5, 3, 2, 7, 6, 4, 2, 9, 1, 2, 6, 8, 2, 3, 4 };
+
+   int array = sizeof(list);
+
+   int Size = sizeof(list[0]);
```

Comment:
```suggestion
   int size = sizeof(list[0]);
```

Make sure that you're consistent with capitalization of variable names